### ISSUE TITLE
 Extended validation SSL certificate is not indicated in browser URL field

### ISSUE THREAD

**Issue by lansana** [brave/brave-browser#3860]
As title says, when viewing a website that has an Extended Validation SSL Certificate, Brave Browser does not indicate this in the URL bar at the top by showing the prefix of the SSL certificate. The only thing Brace Browser shows is a lock icon.

I believe this should be honored as it is a security feature that is helpful in preventing phishing and the likes. The current functionality in Brave Browser strays from the (beneficial) convention of previous browsers.

**Comment by bsclifton** [comment]
@lansana EV not being shown differently than regular SSL is known and intended

Maybe @tomlowenthal or @diracdeltas can comment, but this won't be something that we fix

**Comment by diracdeltas** [comment]
We have gone back and forth on this a bit in the history of Brave. Initially Brave launched without any EV indicators and defended our decision not to in https://github.com/brave/browser-laptop/issues/791. Eventually since there were a few users who weren't employed by the CA industry who asked for it, we did add EV indicators.

We launched the new Brave without EV indicators and haven't heard many complaints yet. It is fairly easy to change so we could reconsider again.
